q2_types/ordination/_transformer.py

```diff
@@ -21,5 +36,15 @@ def _1(data: skbio.OrdinationResults) -> OrdinationFormat:
 
 @plugin.register_transformer
 def _2(ff: OrdinationFormat) -> skbio.OrdinationResults:
-    return skbio.OrdinationResults.read(str(ff), format='ordination',
-                                        verify=False)
+    return _ordination_format_to_ordination_results(ff)
+
+
+@plugin.register_transformer
+def _3(ff: OrdinationFormat) -> pd.DataFrame:
```

Comment:
Can you remove this transformer since it isn't needed right now? If we keep the transformer, we'll be interpreting `OrdinationFormat` sample coordinates as a `DataFrame`, but we may want to open this up to further discussion since the ordination results can be viewed in other ways (e.g. in a feature-centric manner). Having a concrete use-case in the future would help drive the discussion/design, and in the meantime it's really easy to get a `DataFrame` from either `OrdinationResults` or its `Metadata` view.